src/analytics/Properties.tsx

```diff
@@ -1552,7 +1552,12 @@ interface EarnCommonProperties {
 }
 
 interface EarnDepositProperties extends EarnCommonProperties {
-  tokenAmount: string
+  depositTokenAmount: string
+  mode: EarnDepositMode
+  // the below are mainly for swap-deposit. For swap deposit, this would just be
```

Comment:
do you mean "For deposit" (instead of "For swap deposit")?

Reply:
oops, yes